config/sankey_csv.yml

```diff
@@ -1395,6 +1435,27 @@ rows:
     Category: Feedstock
     Type: Input
     Value: imported_ammonia_to_feedstock_in_sankey
+
+  - Group: Primary demand
+    Carrier: Methanol
+    Category: Import
+    Type: Output
+    Value: import_of_methanol
+  - Group: Conversion
+    Carrier: Methanol
+    Category: Synthetic methanol production
+    Type: Output
+    Value: conversion_to_synthetic_methanol
+  - Group: Final demand
+    Carrier: Methanol
+    Category: Export
+    Type: Input
+    Value: methanol_to_export_in_sankey
+  - Group: Final demand
+    Carrier: Methanol
+    Category: Feedstock
+    Type: Input
+    Value: methanol_to_feedstock_in_sankey
```

Comment:
@kndehaan for completeness, shouldn't we add final demand queries for all sectors?

Reply:
Yes, I agree it would be more complete to add final demand queries for all sectors for both methanol and waste mix.